catalog/task/collect-extra-data/0.1/collect-extra-data.yaml

```diff
@@ -0,0 +1,89 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: collect-extra-data
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to collect extra data from release resources
+  params:
+    - name: release
```

Comment:
I thought currently the plan is to only have `extraData` in `ReleasePlanAdmission`. I don't see anything about the other resources in the jira either. We also only have a story for adding the field to RPA and not anywhere else. Is there a reason to support the other resources as well?

Reply:
The three of them will have it.

Reply:
ok, fair enough.